pymbar/timeseries.py

```diff
@@ -972,4 +972,4 @@ def detect_equilibration_binary_search(A_t, bs_nodes=10):
             start = time_grid[k - 1]
             end = time_grid[k + 1]
 
-    return (t, g, Neff_max)
+    return t, g, Neff_max
```

Comment:
Remind me why this change? (Just so I know good formatting practices)

Reply:
They are redundant in this case. You do need to add them for single-element tuple for extra clarity (see [here](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)), but for single expression statements like a `return`, they can (and IMO should) be omitted for n-element tuples.

Reply:
There might be more, now that you bring this up.

Reply:
Yep, four more. All of them in `timeseries`. Do you want me to replace those too?

Reply:
Sure, go for it!